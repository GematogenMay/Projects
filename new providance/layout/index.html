<!DOCTYPE html>
<html>
<head>
	<title>New Providance</title>
	<link href="https://fonts.googleapis.com/css?family=Varela+Round&display=swap" rel="stylesheet">
	<link rel="stylesheet" href="css/main.css">
</head>
<body>
	<header class="header">
		<div class="container">
			<div class="nav">
				<div class="menu1">
					<div class="button_menu">
						<div class="line_button"></div>
						<div class="line_button"></div>
						<div class="line_button"></div>
					</div>
					<div class="menu1_txt">
					<p class="txt_menu">Tour</p>
					<p class="txt_menu">Features</p>
					<p class="txt_menu">Pricing</p>
					</div>
				</div>
				<img src="img/logo.png" class="logo">
				<div class="menu2">
					
				</div>
			</div>
			<div class="offer">
				
			</div>
			<div class="h_iphone">
				
			</div>
		</div>
	</header>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<div class="section">
		<div class="container">
			
		</div>
	</div>
	<footer>
		<div class="container">
			
		</div>
	</footer>

</body>
</html>