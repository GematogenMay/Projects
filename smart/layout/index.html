<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>Smart</title>
	<link rel="stylesheet" type="text/css" href="css/main.css">
	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700&display=swap" rel="stylesheet">
</head>
<body class="body">
	<header class="header clearfix">
		<div class="container">
		<div class="nav">
			<h2><span class="myRedA">S</span>mart</h2>			
		</div>
		<div class="offer">
			<h3><span class="myRedA">Smart</span> Agency Landign Page</h3>
			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Maiores, fuga provident. Libero magni <br>quibusdam labore ad iste saepe hic, maxime qui ipsum tempore beatae, volu tomi ptates <br> doloremque rerum aspernatur dolorum at!</p>
		</div>
		<div class="mail_form">
			<h4 class="mail_title"><span class="myRedA"> Send</span> A Message</h4>
			<form class="mail_input">
				<p>
					<input type="text" name="name" placeholder="Your Name" class="mail_name"><br>
					<input type="text" name="name" placeholder="Your Email" class="mail_email"><br>
					<input type="text" name="name" placeholder="Your Message" class="mail_messege">

				</p>
			</form>
			<div class="mail_button"><button class="mail_submit"><a href="https://mail.google.com/mail/u/0/" class="mail_send">Send Message</a></button></div>
		</div>
		</div>
	</header>
	<section>
		<div class="container">
			<div class="process">
				<div class="title">
					<h3><span class="myRedA">Working </span>Process</h3>
					<p> Lorem ipsum dolor sit amet consectet ur adipiscing elit  Vestibulum bibend um vestibulum. </p>
				</div>
				<div class="process_forms clearfix">
					<div class="process_form1">
						<img src="img/proc1.png" alt="Magnit">
						<h4>Koc Koi Tomi</h4>
						<p>Lorem ipsum is dummy you <br>  lorem very dumyL is </p>
						<button class="process_button clearfix"><a href=# class="process_button_a">Know More</a></button>
					</div>
					<div class="process_form2">
						<img src="img/proc2.png" alt="Bug">	
						<h4>Koimona Set</h4>
						<p>Lorem ipsum is dummy you  <br> lorem very dumyL is </p>
						<button class="process_button clearfix"><a href=# class="process_button_a">Know More</a></button>
					</div>
					<div class="process_form3">
						<img src="img/proc3.png" alt="Setting">
						<h4>Setting Koire</h4>
						<p>Lorem ipsum is dummy you <br>  lorem very dumyL is </p>
						<button class="process_button clearfix"><a href=# class="process_button_a">Know More</a></button>
					</div>
					<div class="process_form4">
						<img src="img/proc4.png" alt="Camera">
						<h4>Ko Image Lara</h4>
						<p>Lorem ipsum is dummy you <br>  lorem very dumyL is </p>
						<button class="process_button clearfix"><a href=# class="process_button_a">Know More</a></button>
					</div>
				</div>
			</div>
		</div>
	</section>
	<section class="featured_works clearfix">
		<div class="container">
			<div class="works clearfix" >
				<div class="title">
					<h3><span class="myRedA">Featured </span> Works</h3>
					<p> Lorem ipsum dolor sit amet consectet ur adipiscing elit  Vestibulum bibend um vestibulum. </p>
				</div>
				<div class="works_img">
						<div class="work_img1">						
						</div>
						<div class="work_img2">						
						</div>
						<div class="work_img3">						
						</div>
							<div class="repeat_img">
							<div class="work_img1">						
							</div>
							<div class="work_img2">						
							</div>
							<div class="work_img3">						
							</div>
							</div>
								<div class="repeat_img">
								<div class="work_img1">						
								</div>
								<div class="work_img2">						
								</div>
								<div class="work_img3">						
								</div>
								</div>						

				</div>			
			</div>
		</div>
	</section>
	<section class="services">
		<div class="container">
			<div class="title">
					<h3><span class="myRedA">Services</span> We Provid</h3>
					<p> Lorem ipsum dolor sit amet consectet ur adipiscing elit  Vestibulum bibend um vestibulum. </p>
			</div>
			<div class="services_item">
			<div class="services_imgs">
				<img src="img/service_diagram.png" alt="Diagram">
				<img src="img/service_settings.png" alt="Settings">
				<img src="img/service_search.png" alt="Search">
				<img src="img/service_diagram.png" alt="Diagram">
			</div>
			<div class="services_txt">
				<h5>Modern App Design</h5>
				<div class="services_border">
					
				</div>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus earum totam ad sint error porro. Consectetur, sunt, aperiam. Magni rem optio distinctio illum consequuntur delectus nulla illo aperiam itaque cumque.
					<br><br>
				Lorem ipsum dolor sit amet, consectetur adipisicing elit. Vel officia vitae eveniet quam rem unde facilis. <br><br>Vitae eveniet quam rem unde facilis. </p>
				<button class="services_button"><a href="#" class="services_butt_txt">Know More</a></button>
			</div>
			<div class="service_img">
			<img src="img/service_apple.png" alt="Apple Watch">
			</div>
			</div>
			</div>
	</section>
	<section class="team">
		<div class="container">
			<div class="title">
					<h3><span class="myRedA">Team</span> Chinen & Skills</h3>
					<p> Lorem ipsum dolor sit amet consectet ur adipiscing elit  Vestibulum bibend um vestibulum. </p>
			</div>
			<div class="team_imgs">
				<div class="team_img1">
					<div class="team_txt">
						<h6>Bedi Manos</h6>
						<p>UI/UX Parena</p>
					</div>						
				</div>
				<div class="team_img2">
					<div class="team_txt">
						<h6>Al Rayhan</h6>
						<p>UI/UX Parena</p>
					</div>										
				</div>
				<div class="team_img3">	
					<div class="team_txt">
						<h6>Ali Sayed</h6>
						<p>UI/UX Parena</p>
					</div>					
				</div>
			</div>
		</div>
	</section>

</body>
</html>