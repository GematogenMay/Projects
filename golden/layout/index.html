<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/main.css">
</head>
<body>


	<header>
		<div class="container">
			
			<div class="heading clearfix">
			<img src="img/logo.png" alt="Golden" class="logo"></img>
			<nav>
				<ul class="menu">
					<li>
						<a href="#">Home</a>
					</li>
					<li>
						<a href="#">Services</a>
					</li>
					<li>
						<a href="#">Portfolio</a>
					</li>
					<li>
						<a href="#">About</a>
					</li>
					<li>
						<a href="#">Contact</a>
					</li>
				</ul>
			</nav>
		</div>
		<div class="titles">
			<div class="titles_first">
				Welcome To Our Studio!
			</div>
		
		<h1>
			It's nice to Meet you
		</h1>
		</div>
				<a class="button" href="#">Tell Me More</a>
		</div>
	</header>


	


	<section>
		<div class="container">
			<div class="title">
				<h2>
					 Services
				</h2>
				<p>
					Proin iaculits purus consequat sem cure
				</p>
			</div>
			<div class="services clearfix">
				
				
				<div class="services_item">
					<img src="img/img1.png" alt="услуга">
					<h3>E-Commerce</h3>
					<p>тут должна быть подпись</p>
				</div>
				
				
				<div class="services_item">
					<img src="img/img2.png" alt="услуга">
					<h3>Responsive Web</h3>
					<p>тут должна быть подпись</p>
				</div>
				<div class="services_item">
					<img src="img/img3.png" alt="услуга">
					<h3>Web-Security</h3>
					<p>тут должна быть подпись</p>
				</div>
			</div>
		</div>
	</section>
	
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>
					 Our Portfolio
				</h2>
				<p>
					Proin iaculits purus consequat sem cure
				</p>
			</div>
			<div class="works clearfix">
				<img src="img/image1.png" alt="работа">
				<img src="img/imag2.png" alt="работа">
				<img src="img/imag4.png" alt="работа">
			</div>
		</div>
	</section>
	


	





	<section>
		<div class="container">
			<div class="title">
				<h2>
					about us
				</h2>
				<p>
					Proin iaculits purus consequat sem cure
				</p>
			</div>
			<div class="about_l">
				<div class="about_l_txt">
				<h3>july 2010 </h3>
				<h3>Our Humble Beginnings</h3>
				<p>Place for text</p>
				</div>
				<img src="img/IM1.png" alt="о нас">
			</div>
			<div class="about_r">
				<div class="about_r_txt">
				<h3>january 2011</h3>
				<h3>Facing Startup Battles</h3>
				<p>Place for text</p>
				</div>
				<img src="img/IM2.png" alt="о нас">
			</div>
			<div class="about_l">
				<div class="about_l_txt">
				<h3>decemer 2012 </h3>
				<h3>Enter The Dark Days</h3>
				<p>Place for text</p>
				</div>
				<img src="img/IM4.png" alt="о нас">
			</div>
			<div class="about_r">
				<div class="about_r_txt">
				<h3>february 2014</h3>
				<h3>Our Trumph</h3>
				<p>Place for text</p>
				</div>
				<img src="img/IM5.png" alt="о нас">
			</div>
		</div>		
</section>
	
<footer>
	<div class="container"> 
		<p>Copyrigth 2014 FreebiesXpress.com</p>
			
	</div>	
</footer>	



</body>





</html>
